src/main/java/com/blackducksoftware/integration/alert/common/AboutReader.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (C) 2018 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.blackducksoftware.integration.alert.common;
+
+import java.nio.charset.StandardCharsets;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.blackducksoftware.integration.alert.web.model.AboutModel;
+import com.blackducksoftware.integration.util.ResourceUtil;
+import com.google.gson.Gson;
+
+@Component
+public class AboutReader {
+    public final static String PRODUCT_VERSION_UNKNOWN = "unknown";
+    private final static Logger logger = LoggerFactory.getLogger(AboutReader.class);
+    private final Gson gson;
+
+    @Autowired
+    public AboutReader(final Gson gson) {
+        this.gson = gson;
+    }
+
+    public AboutModel getAboutModel() {
+        AboutModel aboutModel;
+        try {
+            final String aboutJson = ResourceUtil.getResourceAsString(getClass(), "/about.txt", StandardCharsets.UTF_8.toString());
+            aboutModel = gson.fromJson(aboutJson, AboutModel.class);
+        } catch (final Exception e) {
+            aboutModel = null;
+            logger.error(e.getMessage(), e);
+        }
+        return aboutModel;
```

Comment:
Would it be cleaner to remove the aboutModel object and just return it on line 52 and return null on line 56?